tests/test_schema_structure.py

```diff
@@ -0,0 +1,11 @@
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+
+def test_schema_structure(schemas):
+    metaschema = schemas.pop("metadata.metaschema.1")
+    for key, schema in schemas.items():
+        try:
+            validate(schema, schema=metaschema)
+        except ValidationError as e:
+            raise ValidationError("Schema {} failed validation".format(key)) from e
```

Comment:
It'd be useful to have a test for the defaults here, like checking whether the telemetry ping has the metadata field. This also implies that the tests require the schemas are generated (which we do in CI, but should eventually move into the test suites).

Reply:
> This also implies that the tests require the schemas are generated (which we do in CI, but should eventually move into the test suites).

We already make that assumption. `schemas` here is referencing files under `schemas`.

I will add a test for defaults being applied.